index.html

```diff
@@ -49,23 +53,41 @@ <h2>Autarky is</h2>
     </div>
     <div class='map-overlay' id="info" onclick="hideInfo()">
         <div id="infotext">
-            <h2>Possibility for renewable electricity autarky in Europe</h2>
+            <h1>Can my municipality supply itself with electricity from sun and wind?</h1>
```

Comment:
The length of this new text is a problem on smaller screens, for example my tablet. The old text was ok, just because it was short enough. The new isn't displayed properly. I do not have a technical solution for that however.